cds_ils/migrator/SeriesLoader.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015-2018 CERN.
+#
+# cds-migrator-kit is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS Migrator Records loader."""
+
+import logging
+import uuid
+
+import click
+from invenio_app_ils.errors import IlsValidationError
+from invenio_app_ils.series.api import Series, SeriesIdProvider
+from invenio_db import db
+
+from cds_ils.minters import legacy_recid_minter
+
+cli_logger = logging.getLogger("migrator")
+
+
+class CDSSeriesDumpLoader(object):
```

Comment:
wouldn't it be better to rename `CDSDocumentLoader` to something more generic and use parameters, instead of copying the code to a new class?